RecoTauTag/RecoTau/plugins/PATTauDiscriminationByMVAIsolationRun2.cc

```diff
@@ -120,6 +122,8 @@ namespace reco {
         else if (mvaOpt_ == kDBoldDMwLT || mvaOpt_ == kDBnewDMwLT || mvaOpt_ == kPWoldDMwLT || mvaOpt_ == kPWnewDMwLT ||
                  mvaOpt_ == kDBoldDMwLTwGJ || mvaOpt_ == kDBnewDMwLTwGJ)
           mvaInput_ = new float[23];
+        else if (mvaOpt_ == kDBnewDMwLTwGJPhase2)
+          mvaInput_ = new float[30];
```

Comment:
outside the scope of this PR, but in the future, this should move to using smart pointers (avoid manual memory management) and possibly `std::array`

Reply:
thanks for the comment! We will note that as a clean up ToDo. (@mbluj also for your interest)